### PR TITLE
Adding remark about changing the KeySize property #5307

### PR DESCRIPTION
**Pull request by kapiljaveri**
## Summary

Appended information about changing the KeySize property and its impact on the Key itself to the remarks section.

Let me know if this looks good or if you'd like any changes.

Fixes dotnet/docs#5307

### CODE REVIEW

**Comment by rpetrusha** [comment]
@bartonjs, could you take a look, please?

File: xml/System.Security.Cryptography/AesCryptoServiceProvider.xml [hunk 1]
```diff
@@ -625,6 +625,8 @@
 ## Remarks  
  The minimum size of the key is 128 bits, and the maximum size is 256 bits.  
   
+ Changing the KeySize value will reset the Key itself and cause a new random key to be generated. This will happen any time the setter of the KeySize property is invoked (including when invoked for the same value).
```
Comment (bartonjs) [inline xml/System.Security.Cryptography/AesCryptoServiceProvider.xml]: The technical aspects are accurate, invoking set_KeySize for AesCryptoServiceProvider (or, really, any SymmetricAlgorithm-derived type) will cause the Key property to get randomized to a new value of that size, even when doing `alg.KeySize = alg.KeySize`.

The sentence construction "will reset the Key itself" feels clunky to me, though.

Comment (kapiljaveri) [reply xml/System.Security.Cryptography/AesCryptoServiceProvider.xml]: I can see that. Any suggestions?

Could omit the word "itself"? Or possibly just simplify it to "Changing the KeySize value will cause a new random key to be generated."

**Comment by kapiljaveri** [comment]
Thanks for all the feedback @rpetrusha and @bartonjs! Let me know if the changes look good.

**Comment by rpetrusha** [comment]
Your changes look good, @kapiljaveri. We'll merge your PR now. I should be live on docs.microsoft.com in the next day or two.

**Comment by kapiljaveri** [comment]
Awesome, thank you 😊

**Comment by kapiljaveri** [comment]
Thanks again for all the feedback and patience @bartonjs & @rpetrusha

**Pull request merged**